regression/cloud_providers.yaml

```diff
@@ -0,0 +1,27 @@
+name: streaming-regression-test-cloud-providers
+compute:
+  cluster: r1z1
+gpu_num: 8
+command: |-
+  pip uninstall -y mosaicml-streaming
+  cd streaming
+  pip install -e '.[dev]'
+  composer -n 1 regression/synthetic_dataset.py --create --cloud gs
```

Comment:
I am thinking if we can add a command line arg for the type of dataset ? Soon, we will have to add synthetic computer vision dataset, and maybe others ?

Reply:
Let's do this in a separate PR where we implement CV dataset creation and add it to the yamls/command line args as necessary.